lib/web_ui/lib/src/engine/platform_dispatcher.dart

```diff
@@ -449,12 +451,15 @@ class EnginePlatformDispatcher extends ui.PlatformDispatcher {
         return;
 
       case 'flutter/platform_views':
-        if (useCanvasKit) {
-          rasterizer!.surface.viewEmbedder
-              .handlePlatformViewCall(data, callback);
-        } else {
-          ui.handlePlatformViewCall(data!, callback!);
+        if (_platformViewMessageHandler == null) {
```

Comment:
nit: use `??=`

Reply:
Yes, to me the if is still more legible, but `??=` is better.